01week/rockPaperScissors.js

```diff
@@ -10,18 +10,18 @@ const rl = readline.createInterface({
 function rockPaperScissors(hand1, hand2) {
   let p1 = hand1.toLowerCase().trim();
   let p2 = hand2.toLowerCase().trim();
-  
+
   // Tie games
   if (
     (p1 === "rock" && p2 === "scissors") ||
     (p1 === "paper" && p2 === "rock") ||
     (p1 === "scissors" && p2 === "paper")
-  )
-    return 'Hand one wins!';
-  else if (p1 === p2) {
-    return 'It\'s a tie!';
+  ) {
+    return "Hand one wins!";
+  } else if (p1 === p2) {
+    return "It's a tie!";
   } else {
-    return 'Hand two wins!';
+    return "Hand two wins!";
   }
```

Comment:
Looks like with this logic, if player 1 and player 2 type anything other than [rock,paper,scissors] it will returns Hand two wins!

Reply:
Thank you so much for the comments! I'll check it out and correct it as needed.

Reply:
I added input rules for the game. they ensure that both players must enter 'rock', 'paper', or 'scissors' to play.  

initially, i thought that || would be the operator that would make the condition work, but it was &&. i'm a little fuzzy on why still.  i'll continue to think about it.

Thank you again! I love the feedback!